include/merlin/core_kernels.cuh

```diff
@@ -497,6 +504,9 @@ __forceinline__ __device__ void move_key_to_new_bucket(
           (new_start_idx + tile_offset + src_lane) & (bucket_max_size - 1);
       local_size = buckets_size[new_bkt_idx];
       if (rank == src_lane) {
+        K hashed_key = Murmur3HashDevice(key);
+        uint8_t target_digest = static_cast<uint8_t>(hashed_key >> 32);
```

Comment:
const

Reply:
OK, and I will find all of this and add const, thanks for your review.